README.md

```diff
@@ -10,35 +10,41 @@ game engine in Emacs. It gives syntax highlighting and indentations.
 ![](assets/emacs-gdscript-imenu.png)
 
 <!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
-
 **Table of Contents**
 
-- [Features](#features)
-- [Contributing](#contributing)
-- [How to install](#how-to-install)
-  - [Installing in Spacemacs](#installing-in-spacemacs)
-  - [Installing in Doom Emacs](#installing-in-doom-emacs)
-  - [Installing with `use-package` + `straight.el`](#installing-with-use-package--straightel)
-  - [Installing manually](#installing-manually)
-- [Auto-completion with the Language Server Protocol (LSP)](#auto-completion-with-the-language-server-protocol-lsp)
-- [How to use](#how-to-use)
-  - [Opening the project in the editor](#opening-the-project-in-the-editor)
-  - [Running Godot with visual debug options](#running-godot-with-visual-debug-options)
-  - [Using Hydra](#using-hydra)
-  - [Formatting code with gdformat](#formatting-code-with-gdformat)
-  - [Browsing the Godot API with eww](#browsing-the-godot-api-with-eww)
-- [Keyboard shortcuts](#keyboard-shortcuts)
-- [Customization](#customization)
-- [Using the debugger](#using-the-debugger)
-  - [Adding and removing breakpoints](#adding-and-removing-breakpoints)
-  - [Running the project with the debugger active](#running-the-project-with-the-debugger-active)
-  - [Fetching an object's details](#fetching-an-objects-details)
-  - [Debug Hydra](#debug-hydra)
-  - [The `* Stack frame vars *` buffer](#the--stack-frame-vars--buffer)
-  - [`* Inspector *` buffer](#-inspector--buffer)
-  - [`* Stack dump *` buffer](#-stack-dump--buffer)
-  - [`* Breakpoints *` buffer](#-breakpoints--buffer)
-  - [`* Scene tree *` buffer](#-scene-tree--buffer)
+- [GDScript mode for Emacs](#gdscript-mode-for-emacs)
```

Comment:
Better if we remove the root level? So we keep the same style.

Reply:
Hi, @jcs090218 I am not very familiar with the configuration of markdown-doc.

It uses the default format and there are no formatting requirements in this `README.MD` .I have now removed the H1 title. May I ask if there is still a problem

Reply:
@jcs090218 `markdown-toc` has a bug [issuse/53](https://github.com/ardumont/markdown-toc/issues/53). 
Should I manually edit it?

Reply:
I have used the following configuration and can now automatically generate directories normally with `markdown-toc`.

```emacs-lisp
(with-eval-after-load 'markdown-toc
  (custom-set-variables '(markdown-toc-user-toc-structure-manipulation-fn 'cdr))
  (setq markdown-toc-indentation-space 2)
  (defun markdown-toc--compute-toc-structure (imenu-index)
    "Given a IMENU-INDEX, compute the TOC structure."
    (--mapcat (markdown-toc--compute-toc-structure-from-level
               (if (eq markdown-toc-user-toc-structure-manipulation-fn 'cdr)
                   -1
                 0)
               it)
              imenu-index)))
```